client/my-sites/hosting/github/github-connect-card/use-github-branches.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from 'react-query';
+import wp from 'calypso/lib/wp';
+
+const USE_GITHUB_BRANCHES_QUERY_KEY = 'github-branches-query-key';
+const CACHE_TIME = 1000 * 60 * 5; // 5 mins
```

Comment:
I don't think branches are changed often and for deploying most times we gonna need master/main branch. I think it's a good candidate for caching, Can't scientifically justify 5 mins, I just thought it would be a good buffer in case user modify their branches. Would you recommend something higher like say 24hr?

Reply:
Nah I think I'd just leave it as the default.